articles/Third Party Tools and Libraries/enable-math.md

```diff
@@ -47,7 +58,7 @@ Frist delete the following folders:
 
 Then in the `mathjax/config/` folder, delete all files but `TeX-AMS-MML_HTMLorMML-full.js`, which is the only config file used in TAO.
 
-An already shrunk Mathjax archive is available [here](../resources/mathjax-shrinked.zip).
+An already shrunk Mathjax archive is available [here](../resources/third-party/mathjax-shrinked.zip).
```

Comment:
Spelling: mathjax-shrinked should also be shrunk (http://www.verbix.com/webverbix/English/shrink.html)